### DISCUSSION TITLE
How to achive that

### DISCUSSION THREAD

**Discussion by prashantdwivedi** [squidfunk/mkdocs-material#4254] [Q&A] [answered]
Hi, please let me know what to do to achieve that.
add an icon not show!
![CleanShot 2022-08-16 at 00 21 03@2x](https://user-images.githubusercontent.com/6306052/184697443-4e6688b4-dcef-4334-8169-352a47d4cf3e.png)
Code view
<img width="485" alt="CleanShot 2022-08-16 at 00 22 19@2x" src="https://user-images.githubusercontent.com/6306052/184697585-04687ba5-a7cc-4478-b865-1457ba8c8e42.png">
Preview 
<img width="293" alt="CleanShot 2022-08-16 at 00 22 42@2x" src="https://user-images.githubusercontent.com/6306052/184697693-804e13aa-2eb1-4b8e-9244-a8be4c881e77.png">

**Comment by squidfunk** [comment, marked as answer]
[Navigation icons](https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-icon) are currently incubating in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/), which means it's [reserved to sponsors](https://squidfunk.github.io/mkdocs-material/insiders/#whats-in-for-me). If you sponsor @squidfunk with **$10/month or more**, you'll get access to Insiders and can use this feature immediately.

Otherwise, you'll have to wait [until the funding goal is reached](https://squidfunk.github.io/mkdocs-material/insiders/#12000-piri-piri).

**Reply by prashantdwivedi** [reply]
Oh, I got it thanks for the information.
